dask_sql/utils.py

```diff
@@ -134,44 +130,23 @@ def __str__(self):
 
 
 def convert_sql_kwargs(
-    sql_kwargs: Dict["LogicalPlan", "LogicalPlan"],
+    sql_kwargs: Dict[str, str],
 ) -> Dict[str, Any]:
     """
-    Convert a HapMap (probably coming from a SqlKwargs class instance)
-    into its python equivalent. Basically calls convert_sql_kwargs
-    for each of the values, except for some special handling for
-    nested key-value parameters, ARRAYs etc. and CHARs (which unfortunately have
-    an additional "'" around them if used in convert_sql_kwargs directly).
+    Convert the Rust Vec of key/value pairs into a Dict containing the keys and values
     """
 
-    def convert_literal(value):
-        if isinstance(value, "LogicalPlan"):
-            operator_mapping = {
-                "ARRAY": list,
-                "MAP": lambda x: dict(zip(x[::2], x[1::2])),
-                "MULTISET": set,
-                "ROW": tuple,
-            }
```

Comment:
Out of scope, but I realize we really aren't testing any of these operator mappings in our create table with tests - might make sense to add those in later on if we intend to add back support for these operators in a follow up